### PR TITLE
Bone hotfix

### PR DESCRIPTION
**Pull request by MadmanMartian**
Fixes bones being made of paper
## About The Pull Request

Previously, due to the inline and the compiler being dumb, rather than doing math, it would return 1 instead, so your bones would effectively have 1HP
## Why It's Good For The Game

People can't explode into a million pieces from slipping on a banana peel

### CODE REVIEW

**Comment by genessee596** [comment]
This was arguably one of the funniest bugs to witness firsthand on a test server - your buddy's skull exploding when you threw a bruise pack at him wasn't a moment I'll soon forget.

Thank you for fixing this before server launch.

**Review by TheShown911 — APPROVED** [review]

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

**Comment by genessee596** [comment]
Got approval from maintainer and delegate. Merging now.

**Pull request merged**
